language/move-prover/boogie-backend/src/spec_translator.rs

```diff
@@ -47,6 +47,8 @@ pub struct SpecTranslator<'env> {
     /// Set of items which have been already traced. This is used to avoid redundant tracing
     /// of expressions whose value has been already tracked.
     traced_items: RefCell<BTreeSet<TraceItem>>,
+    /// Local variables with a known translation.
+    assigned_local_vars: RefCell<HashMap<Symbol, Vec<String>>>,
```

Comment:
We uniformly use BTreeX collections in the code base (of Diem even I think) because of the potential non-determinism of HashMap. A map which is never enumerated like this one should be save. But as far as performance goes, its also interesting whether a map which has 2-3 elements in average might be faster as BTree.

Reply:
Sure I can switch to BTreeMap